### ISSUE TITLE
Cypress cannot find browser with Chrome 80

### ISSUE THREAD

**Issue by katz12** [cypress-io/cypress#6334]
### Current behavior:

All of our cypress tests in CI are failing with:

```
The automation client disconnected. Cannot continue running tests.

Timed out waiting for the browser to connect. Retrying...
The automation client disconnected. Cannot continue running tests.

Timed out waiting for the browser to connect. Retrying again...
The automation client disconnected. Cannot continue running tests.

The browser never connected. Something is wrong. The tests cannot run. Aborting...
```

### Desired behavior:

Previous runs passed without issue on Chrome 79 and below.
The first test that ran on Chrome 80 failed.

### Test code to reproduce

We are using https://github.com/cypress-io/cypress-docker-images/blob/master/browsers/chrome69/Dockerfile which seems to download the latest Chrome every time. Up until this morning it was installing Chrome 79 and now it is installing Chrome 80.

Any test run in this environment fails.

### Versions

Cypress 3.6.1
Chrome 80

**Comment by cellog** [comment]
```
/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/MacOS/Cypress[74243]: ../../third_party/electron_node/src/node_platform.cc:444:std::shared_ptr<PerIsolatePlatformData> node::NodePlatform::ForIsolate(v8::Isolate *): Assertion `data' failed.
 1: 0x110ba8f85 node::Abort() [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 2: 0x110ba8d0f node::Assert(node::AssertionInfo const&) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 3: 0x110c10407 node::options_parser::SplitHostPort(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > >*) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 4: 0x10c12aaaf v8::internal::Isolate::SetUseCounterCallback(void (*)(v8::Isolate*, v8::Isolate::UseCounterFeature)) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 5: 0x10c16d9b8 v8::internal::IncrementalMarking::V8Step(double, v8::internal::IncrementalMarking::CompletionAction, v8::internal::StepOrigin) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 6: 0x10c1603aa v8::internal::IncrementalMarking::AdvanceOnAllocation() [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 7: 0x10c16003e v8::internal::IncrementalMarking::Observer::Step(int, unsigned long, unsigned long) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 8: 0x10bf48e3b v8::internal::Heap::KeepDuringJob(v8::internal::Handle<v8::internal::JSReceiver>) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
 9: 0x10c1a87e2 v8::internal::NewSpace::EnsureAllocation(int, v8::internal::AllocationAlignment) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
10: 0x10bf4330b v8::internal::Heap::PromotedExternalMemorySize() [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
11: 0x10bf43111 v8::internal::Heap::PromotedExternalMemorySize() [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
12: 0x10bf43511 v8::internal::Heap::PromotedExternalMemorySize() [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
13: 0x10c14538a v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
14: 0x10c3ceadf v8::internal::RegExp::CompileForTesting(v8::internal::Isolate*, v8::internal::Zone*, v8::internal::RegExpCompileData*, v8::base::Flags<v8::internal::JSRegExp::Flag, int>, v8::internal::Handle<v8::internal::String>, v8::internal::Handle<v8::internal::String>, bool) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
15: 0x10ca10499 v8::internal::compiler::ZoneStats::ReturnZone(v8::internal::Zone*) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
16: 0x10ca7a558 v8::internal::compiler::ZoneStats::ReturnZone(v8::internal::Zone*) [/Users/gbeaver/Library/Caches/Cypress/3.8.3/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework]
The Test Runner unexpectedly exited via a exit event with signal SIGABRT

Please search Cypress documentation for possible solutions:

https://on.cypress.io

Check if there is a GitHub issue describing this crash:

https://github.com/cypress-io/cypress/issues

Consider opening a new issue.

----------

Platform: darwin (19.2.0)
Cypress Version: 3.8.3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /Users/gbeaver/.nvm/versions/node/v10.17.0/bin/node
Arguments: /Users/gbeaver/.nvm/versions/node/v10.17.0/lib/node_modules/yarn/lib/cli.js run cypress
Directory: /Users/gbeaver/citrine/shire/ui/gollum
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
```
 Cypress 3.8.3

**Comment by brian-mann** [comment]
This is happening due to our Docker images updating (due to a bug in DockerHub) which accidentally updated about 5 images to Chrome 80. 

The issue for that is here: https://github.com/cypress-io/cypress/issues/6335

Chrome 80 has a separate problem in Cypress where the automation sockets are unsuccessfully connecting.

We're working to restore the previous images to the correct version of Chrome which will fix this problem.

I'm closing as a duplicate but will add a comment when the images are fixed.

**Comment by cellog** [comment]
@brian-mann is there an issue to track for the automation sockets? We are seeing a flat out crash of chrome about 2/3 through running all specs which is killing our productivity. Fortunately, electron works but is much much slower

**Comment by brian-mann** [comment]
@cellog we don't have any images with chrome 80 - and have not yet investigated the root cause. We suggest you use chrome 79 in CI for the time being, or download a fixed version of Chromium locally.

We're in the middle of releasing Cypress 4.0, and then we'll look into the Chrome 80 problems.

The issue regarding the Docker containers (which have been fixed) is here: https://github.com/cypress-io/cypress/issues/6335

**Comment by katz12** [comment]
Thanks for the responses everyone.

Interestingly we are finding that updating to Cypress 3.8.3 is resulting in passing tests. Although this seems odd given the issues @cellog is having.

**Comment by cotufaloschiflones** [comment]
@katz12 I solved it same as you, with version 3.8.3, to be specific `cypress/included:3.8.3`.

I'm on a mac computer and following the [doc from docker](https://docs.docker.com/docker-for-mac/networking/),  I can make the tests pass consistently with this command:
`docker run -it -v $PWD:/e2e -w /e2e -e CYPRESS_baseUrl=http://host.docker.internal:8080 cypress/included:3.8.3 --env ABORT_ON_FAIL=true`

In my CI (Travis) I run the next command since it is linux:
`docker run -it --ipc host --network host -v $PWD:/e2e -w /e2e -e CYPRESS_baseUrl=http://localhost:8080 cypress/included:3.8.3 --record --key $cypress_key --env ABORT_ON_FAIL=true`

I hope this can help!
